diff -up breathe-4.34.0/setup.py.omv~ breathe-4.34.0/setup.py
--- breathe-4.34.0/setup.py.omv~	2023-01-12 20:01:34.081351351 +0100
+++ breathe-4.34.0/setup.py	2023-01-12 20:01:41.208065949 +0100
@@ -17,7 +17,7 @@ Breathe is an extension to reStructuredT
  render `Doxygen <http://www.doxygen.org>`__ xml output.
 """
 
-requires = ["Sphinx>=4.0,<6,!=5.0.0", "docutils>=0.12"]
+requires = ["Sphinx>=4.0,<7,!=5.0.0", "docutils>=0.12"]
 
 if sys.version_info < (3, 6):
     print("ERROR: Sphinx requires at least Python 3.6 to run.")
